integration-test/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -320,14 +320,14 @@ private[spark] object KubernetesSuite {
   val TEST_SECRET_VALUE = "test-data"
   val TEST_SECRET_MOUNT_PATH = "/etc/secrets"
 
-  val FILE_DOWNLOAD_PATH = "/var/spark-data/spark-files"
+  val IN_POD_FILE_DOWNLOAD_PATH = "/var/spark-data/spark-files"
```

Comment:
Ok. How about CONTAINER_LOCAL_FILE_DOWNLOAD_PATH instead since we already have other names like CONTAINER_LOCAL_SPARK_DISTRO_EXAMPLES_JAR?

Reply:
Done.